content/applications/hr/payroll.rst

```diff
@@ -229,8 +406,8 @@ regular pay, as well as a structure for an `End of Year Bonus` which includes th
 end of year bonus. Both the `Regular Pay` structure and `End of Year Bonus` structure are structures
 within the `Employee` structure type.
 
-The different structure types can be seen by going to :menuselection:`Payroll --> Configuration -->
-Salary --> Structure Types`.
+The different structure types can be seen by navigating to :menuselection:`Payroll app -->
+Configuration --> Salary --> Structure Types`.
 
 There are two default structure types configured in Odoo: *Employee* and *Worker*. Typically,
 *Employee* is used for salaried employees, which is why the wage type is *Monthly Fixed Wage*, and
```

Comment:
```suggestion
There are two default structure types configured in Odoo: :guilabel:`Employee` and
:guilabel:`Worker`.

Typically, :guilabel:`Employee` is used for salaried employees, which is why the :guilabel:`Wage
Type` is :guilabel:`Monthly Fixed Wage`.

:guilabel:`Worker` is typically used for employees paid by the hour, so the :guilabel:`Wage Type` of
:guilabel:`Hourly Wage` is used. 
```
I included the line (currently in 414) in this comment, as it was greyed-out, and unable to leave a comment on. Just as a blanket note: make sure that anything mentioned about something that appears on the UI has a GUI label on it. Thanks! 👍